src/components/form-sections/HasNextEnrollment/index.js

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from 'react';
+import { isEmpty } from 'lodash';
+import { getNextEnrollment } from '../../../lib/services';
+
+const HasNextEnrollment = ({ enrollmentId }) => {
+  const [nextEnrollment, setNextEnrollment] = useState(false);
+
+  useEffect(() => {
+    async function fetchNextEnrollment() {
+      if (!enrollmentId) return setNextEnrollment(false);
+
+      const fetchedNextEnrollment = await getNextEnrollment(enrollmentId);
+
+      setNextEnrollment(fetchedNextEnrollment);
+    }
+
+    fetchNextEnrollment();
+  }, [enrollmentId]);
+
+  if (isEmpty(nextEnrollment)) {
+    console.log('caca');
+    return null;
+  }
+
+  const nextEnrollementId = nextEnrollment[0].id;
+  const nextEnrollementTargetAPI = nextEnrollment[0].target_api;
+
+  return (
+    <div className="notification info">
+      Cette demande est liée à la demande {nextEnrollementTargetAPI || ''}{' '}
```

Comment:
This is a wonderful occasion to use TARGET_API_LABELS from '../../../lib/api':
```suggestion
      Cette demande est liée à la demande {TARGET_API_LABELS[nextEnrollementTargetAPI] || ''}{' '}
```